projects/ngx-editor/src/lib/Editor.ts

```diff
@@ -21,6 +22,7 @@ interface Options {
   schema?: Schema;
   plugins?: Plugin[];
   nodeViews?: EditorProps['nodeViews'];
+  enterKeyHint?: EnterKeyHint;
```

Comment:
```suggestion
  attributes?: Record<string, string>;
```

we can get entire attributes here for forward it to prosemirror.

Reply:
OK, why not. I will change the feature.

Reply:
Done.